src/ApparentHorizons/Tags.hpp

```diff
@@ -13,6 +13,14 @@
 #include "Utilities/TMPL.hpp"
 
 class DataVector;
+class FastFlow;
+
+namespace Tags {
```

Comment:
Why did you resolve this @kidder? May be deferred to #874 but since the namespace is added in this PR anyway for the compute tags, might as well add it here, too.